frontend/src/app/testResults/MultiplexResultInputForm.tsx

```diff
@@ -0,0 +1,280 @@
+    ) {
+      return true;
+    }
+
+    return false;
+  };
+
+  const onResultSubmit = (event: React.FormEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    onSubmit();
+  };
+
+  return (
+    <form className="usa-form maxw-none">
+      <div className="grid-row">
+        <div className="grid-col-4">
+          <h4 className="prime-radio__title">COVID-19</h4>
+          <RadioGroup
+            legend="COVID-19 result"
+            legendSrOnly
+            onChange={(value) => {
+              setDiseaseResult("covid", value);
+            }}
+            buttons={[
+              {
+                value: COVID_RESULTS.POSITIVE,
+                label: `${TEST_RESULT_DESCRIPTIONS.POSITIVE} (+)`,
+              },
+              {
+                value: COVID_RESULTS.NEGATIVE,
+                label: `${TEST_RESULT_DESCRIPTIONS.NEGATIVE} (-)`,
+              },
+            ]}
+            name={`covid-test-result-${queueItemId}`}
+            selectedRadio={resultsMultiplexFormat.covid}
+            wrapperClassName="prime-radio__group"
+            disabled={isSubmitDisabled}
+          />
+        </div>
+        <div className="grid-col-4">
+          <h4 className="prime-radio__title">Flu A</h4>
+          <RadioGroup
+            legend="Flu A result"
+            legendSrOnly
+            onChange={(value) => {
+              setDiseaseResult("fluA", value);
+            }}
+            buttons={[
+              {
+                value: COVID_RESULTS.POSITIVE,
+                label: `${TEST_RESULT_DESCRIPTIONS.POSITIVE} (+)`,
+              },
+              {
+                value: COVID_RESULTS.NEGATIVE,
+                label: `${TEST_RESULT_DESCRIPTIONS.NEGATIVE} (-)`,
+              },
+            ]}
+            name={`flu-a-test-result-${queueItemId}`}
+            selectedRadio={resultsMultiplexFormat.fluA}
+            wrapperClassName="prime-radio__group"
+            disabled={isSubmitDisabled}
+          />
+        </div>
+        <div className="grid-col-4">
+          <h4 className="prime-radio__title">Flu B</h4>
+          <RadioGroup
+            legend="Flu B result"
+            legendSrOnly
+            onChange={(value) => {
+              setDiseaseResult("fluB", value);
+            }}
+            buttons={[
+              {
+                value: COVID_RESULTS.POSITIVE,
+                label: `${TEST_RESULT_DESCRIPTIONS.POSITIVE} (+)`,
+              },
+              {
+                value: COVID_RESULTS.NEGATIVE,
+                label: `${TEST_RESULT_DESCRIPTIONS.NEGATIVE} (-)`,
+              },
+            ]}
+            name={`flu-b-test-result-${queueItemId}`}
+            selectedRadio={resultsMultiplexFormat.fluB}
+            wrapperClassName="prime-radio__group"
+            disabled={isSubmitDisabled}
+          />
+        </div>
+      </div>
+      <div className="prime-test-result-submit grid-row">
+        <div className="grid-col-fill">
+          <Checkboxes
```

Comment:
Noticed some strange behavior with this checkbox unlike other checkboxes. When you click "Inconclusive" and try to uncheck it, it never unchecks. I created a screen recording. Let me know if this is intended behavior.

https://user-images.githubusercontent.com/20211771/175313952-ee63bd71-95c3-4df6-9768-3fa88bb8c7d9.mov

Reply:
That is a weird behavior, but it is not a bug. What is happening is that the backend does not support setting back the results to a null state. So basically when you are trying to uncheck the box you are setting all the results back to null (backend only accepts positive, negative, undetermined mutations) so the request kinda gets ignored. 

We should talk about this today in eng standup @emmastephenson

Reply:
Oh I see. Thank you for letting me know the cause of this issue, Johanna! 

If it helps with the discussion, the behavior for correcting an inconclusive test looks like this 😓 :

https://user-images.githubusercontent.com/20211771/175322654-12211847-ea15-4b05-b8dc-7ac73422955e.mov

Reply:
that does look like a bug, will check into it

Reply:
It seems that after the results are corrected all the possible values get returned back to the frontend. I am not sure if this is the behavior that is expected.
![Screen Shot 2022-06-23 at 10 32 35 AM](https://user-images.githubusercontent.com/103958711/175325190-043cdc05-96b1-4121-8726-aaa1e9664c68.png)

Then depending on what is read first on the results array the front end displays what is told.

Reply:
another weird edge case is if a user is in the middle of correcting an inconclusive test and go to mark a result for another test in the queue 🧐 

https://user-images.githubusercontent.com/20211771/175327552-d0773797-6672-4737-8057-6421ef700f9a.mov